### ISSUE TITLE
`scoverage.coverage` file not generated with `-coverage-out` flag for Scala 3

### ISSUE THREAD

**Issue by armanbilge** [VirtusLab/scala-cli#1072]
**Version(s)**

`0.1.6`

**Describe the bug**

Running the Scala 3 compiler with coverage enabled does not generate the `scoverage.coverage` file.

**To Reproduce**

```scala
//> using scala "3.2.0-RC1-bin-20220604-13ce496-NIGHTLY"
//> using options  "-Ycheck:instrumentCoverage", "-coverage-out:.", "-sourceroot:."

@main def main = ()
```

**Expected behaviour**
Compiling this should generate a `scoverage.coverage` file.

**Comment by lwronski** [comment]
Hi @armanbilge,
it should work when you pass the absolute path  to `-coverage-out` option.

```
$ pwd
/tmp/scala-demo
$ cat Hello.scala 
//> using scala "3.2.0-RC1-bin-20220604-13ce496-NIGHTLY"
//> using options  "-Ycheck:instrumentCoverage", "-coverage-out:/tmp/scala-demo", "-sourceroot:/tmp/scala-demo"

@main def main = ()
$ scala-cli compile Hello.scala 
Compiling project (Scala 3.2.0-RC1-bin-20220604-13ce496-NIGHTLY, JVM)
checking /private/tmp/scala-demo/Hello.scala after phase instrumentCoverage
Compiled project (Scala 3.2.0-RC1-bin-20220604-13ce496-NIGHTLY, JVM)
$ ls
Hello.scala		scoverage.coverage
```

**Comment by armanbilge** [comment]
Thanks! So is a relative path unsupported for scala-cli? It works with sbt.

I thought it may be a similar issue to https://github.com/VirtusLab/scala-cli/issues/530#issuecomment-1081617432 where files are not being copied to the output dir.

**Comment by lwronski** [comment]
yeah, it is a similar problem, but in this case `scoverage.coverage` is generated under ` /Users/$user/Library/Caches/ScalaCli/bloop/` directory for MacOS. 

IMO `scala-cli` should also support a relative path for `scoverage.coverage`

**Comment by lwronski** [comment]
It should be fixed in the next version of scala-cli `0.1.8`. But it required some additional work to handling properly path in using directive, more detail in #1098
